compat/elementor/elementor.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+class SiteOrigin_Widgets_Bundle_Elementor {
+
+	/**
+	 * Get the singleton instance
+	 *
+	 * @return SiteOrigin_Widgets_Bundle_Elementor
+	 */
+	public static function single() {
+		static $single;
+
+		return empty( $single ) ? $single = new self() : $single;
+	}
+
+	private $plugin;
+
+	function __construct() {
+		add_action( 'template_redirect', [ $this, 'init' ] );
+
+		add_action( 'wp_ajax_elementor_render_widget', [ $this, 'ajax_render_widget_preview' ] );
+		add_action( 'wp_ajax_elementor_editor_get_wp_widget_form', [ $this, 'ajax_render_widget_form' ] );
+	}
+
+	function init() {
+
+		$this->plugin = Elementor\Plugin::instance();
```

Comment:
What happens here, in the rare case that a user is using PHP 5.2, plus has Elementor and Widgets Bundle enabled. I know Elementor has some code to prevent itself from loading - does that also stop our compatibility code from loading?